opm/input/eclipse/Parser/Parser.cpp

```diff
@@ -1410,6 +1410,17 @@ bool parseState( ParserState& parserState, const Parser& parser ) {
                                       includeFile.value().generic_string());
                 parserState.loadFile(includeFile.value());
             }
+
+            if (firstRecord.size() > 1) {
+                std::string ignored;
+                for(std::size_t i= 1; i < firstRecord.size(); ++i) {
+                    ignored += readValueToken<std::string>(firstRecord.getItem(i)) + " ";
+                }
+                const std::string msg = fmt::format("The keyword " + rawKeyword->getKeywordName() +
```

Comment:
If you're using libfmt anyway, there's not reason to perform this kind of string concatenation.  Just use the `{}` place holder and pass the keyword name as an argument to `fmt::format()`.